src/dso_api/dynamic_api/views.py

```diff
@@ -338,6 +339,13 @@ def get_queryset(self):
         return self.model.objects.all()
 
     def get_serializer_class(self):
+        def _to_representation(self, instance):
+            ret = super(self.__class__, self).to_representation(instance)
+            for field_name in self.fields.keys():
+                field_value = getattr(instance, field_name)
+                if isinstance(field_value, GEOSGeometry):
+                    ret[field_name] = field_value.wkt
+            return ret
```

Comment:
Je zou dit ook in een `BaseCSVSerializer` class kunnen zetten. Dan heb je minder inline-functies. (net als DynamicModelSerializer, etc.)